Utils/Dataflow/016_task2es/task2es.py

```diff
@@ -2,18 +2,17 @@
 """
 DKB Dataflow stage 016 (task2es)
 
-Transform task metadata (from ProdSys) to fit ES scheme.
+Update task metadata with service fields necessary for ES indexing.
 
 Authors:
   Marina Golosova (marina.golosova@cern.ch)
+  Vasilii Aulov (vasilii.aulov@cern.ch)
 """
 
 import os
 import sys
 import traceback
```

Comment:
```suggestion

```

Reply:
Seems like it is not used either.

Reply:
Done in 11280e7 instead of committing a suggestion - 017 had the same issue as well.